internal/store/builder.go

```diff
@@ -178,7 +178,7 @@ func (b *Builder) WithCustomResourceStoreFactories(fs ...customresource.Registry
 	for i := range fs {
 		f := fs[i]
 		if _, ok := availableStores[f.Name()]; ok {
-			klog.Warningf("The internal resource store named %s already exists and is overridden by a custom resource store with the same name, please make sure it meets your expectation", f.Name())
```

Comment:
Is this because we don't have a way to log warn messages with structure logging?

Reply:
Yes, that does not exist there. See also: https://github.com/kubernetes/klog/pull/184